Repositories/DbRepository.cs

```diff
@@ -79,26 +79,26 @@ public async Task<TEntity> GetByIdAsync(int id)
 
         public void Update(TEntity entity)
         {
-            _context.Update(entity);
-            _context.SaveChanges();
-           /*  var DbEntity = entities.Find(entity.Id);
+            var DbEntity = entities.Find(entity.Id);
             if (DbEntity != null)
             {
                 _context.Entry(DbEntity).CurrentValues.SetValues(entity);
                 _context.SaveChanges();
-            } */
+            }
```

Comment:
Why not use _context.Update() instead? It automatically finds the column with the matching Entity Id and replaces the fields that have changed

Reply:
Well, I tried to use the update method, but it threw an exception that the entity is already being tracked.

Reply:
Okay, I tried it myself, the Update() function doesn't work on our listings 😆

Reply:
Yeah, that was my bad for suggesting it in the previous pull request.
Also, as far as I understand, this line with CurrentValues.SetValues should be unnecessary as SaveChanges should do it for you.